From efbb87f805fcf4325060c9dbfd6c92c01537886c Mon Sep 17 00:00:00 2001
From: Guillem Jover <gjover@sipwise.com>
Date: Thu, 29 Dec 2022 10:14:14 +0100
Subject: MT#55831 Build the library as shared

Build the library as shared.
Otherwise we are trying to link a static library built as PIE into a
shared library.

(real ticket number: TT#6850)

---

--- ngcp-sems-1.6.0.orig/apps/rtmp/librtmp/Makefile
+++ ngcp-sems-1.6.0/apps/rtmp/librtmp/Makefile
@@ -30,7 +30,7 @@ SO_posix=so.0
 SO_mingw=dll
 SO_EXT=$(SO_$(SYS))
 
-SHARED=no
+SHARED=yes
 SODEF_yes=-fPIC
 SOLIB_yes=librtmp.$(SO_EXT)
 SOINST_yes=install_$(SO_EXT)
